includes/discovery/sensors/power/hpblmos.inc.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if ($device['os'] == 'hpblmos') {
+
+    $sensor_type = 'hpblmos_psu_usage';
+    $psu_exists_oid = '.1.3.6.1.4.1.232.22.2.5.1.1.1.16.';
+    $psu_usage_oid = '.1.3.6.1.4.1.232.22.2.5.1.1.1.10.';
+    $psu_max_usage_oid = '.1.3.6.1.4.1.232.22.2.5.1.1.1.9.';
+
+    for ($psuid = 1; $psuid < 7; $psuid++) {
```

Comment:
Could you use snmpwalk to determine which PSUs and fans exist rather than individual gets?